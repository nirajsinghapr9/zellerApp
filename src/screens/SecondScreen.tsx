import React from 'react';
import {View, Text, StyleSheet} from 'react-native';

export default function SecondScreen() {
  return (
    <View style={styles.container}>
      <Text style={styles.text}>This is the second screen (empty).</Text>
    </View>
  );
}

const styles = StyleSheet.create({
  container: {flex: 1, justifyContent: 'center', alignItems: 'center'},
  text: {fontSize: 18, color: '#333'},
});
