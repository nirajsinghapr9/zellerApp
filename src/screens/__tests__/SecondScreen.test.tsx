import React from 'react';
import {render, screen} from '@testing-library/react-native';
import SecondScreen from '../SecondScreen';

describe('SecondScreen', () => {
  it('renders correctly', () => {
    const {toJSON} = render(<SecondScreen />);

    expect(screen.getByText('This is the second screen (empty).')).toBeTruthy();

    expect(toJSON()).toMatchSnapshot();
  });
});
