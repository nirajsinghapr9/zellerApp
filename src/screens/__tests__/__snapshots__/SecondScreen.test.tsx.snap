// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SecondScreen renders correctly 1`] = `
<View
  style={
    {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text
    style={
      {
        "color": "#333",
        "fontSize": 18,
      }
    }
  >
    This is the second screen (empty).
  </Text>
</View>
`;
